src/machine/memory/tlb/tlb.h

```diff
@@ -96,6 +100,7 @@ class TLB : public FrontendMemory {
     TLBType type;
     const TLBConfig tlb_config;
     uint32_t current_satp_raw = 0;
+    CSR::PrivilegeLevel current_priv_ = CSR::PrivilegeLevel::MACHINE;
```

Comment:
The logic should be solved such way, that this field is not needed.

Reply:
There can be use for keeping some last access privilege level and ASID or something similar for visualization purposes. But for sure not for real work.